ui/js/models/ph_series.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
```

Comment:
Ok, I will commit another PR for this change. Then I can pull and rebase it after you merge it :)